### DISCUSSION TITLE
amd64 lima vm for arm64 host / Mac M1

### DISCUSSION THREAD

**Discussion by electriquo** [rancher-sandbox/rancher-desktop#1555] [General]
Lima VM is used as the host machine for containers runtime. By default, it seems that Lima provision a VM with the same platform architecture as the host machine, which Rancher Desktop is installed. E.g. for a host machine with amd64 platform architecture, Lima provision a VM of the same platform architecture

```shell
$ uname -m
x86_64

$ limactl shell 0 uname -m
x86_64
```

Which in turn, spins up containers that are built with the same architecture type of the Lima VM architecture.
For host machine with arm64, it becomes problematic, since not all images are built for multi-architecture platform and usually built for amd64 platform architecture. For instance, postgres has multi-architecture platform images

```shell
$ docker manifest inspect --verbose postgres | jq -r '.[].Descriptor.platform.architecture'
amd64
arm
arm
arm64
386
mips64le
ppc64le
s390x
```

while mysql does not

```
$ docker manifest inspect --verbose mysql | jq -r '.[].Descriptor.platform.architecture'
amd64
```

[Can we instruct Lima VM to provision an amd64 VM on arm64 host machine?](https://github.com/lima-vm/lima/blob/master/docs/multi-arch.md)
Will it solve a host machine with arm64 platform architecture to allow working with containers seamlessly?
How do you advise to do it?

**Comment by jandubois** [comment]
Please see #1501 for the current status quo

**Comment by polsson12** [comment]
Is there any update on this issue?

This must be a huge problem for all the developers and other people wanting to run a dev kubernetes on their M1/M2/M3 Apple computers (including me)
